content/fast/replace-gifs-with-videos/codelab-replace-gifs-with-video.md

```diff
@@ -98,13 +90,13 @@ it makes sense to generate both. Thankfully the `<video>` element will let you
 add multiple sources, so if a browser doesn't support WebM, it can fallback to
 MP4.
 
-- In the console, run:
+1. In the console, run:
 
 <pre class="devsite-terminal devsite-click-to-copy">
 ffmpeg -i cat-herd.gif -c vp9 -b:v 0 -crf 41 cat-herd.webm
 </pre>
 
-- To check the file sizes run:
+2. To check the file sizes run:
```

Comment:
oh oops I think this will end up displaying "1" next to each step because they're not next to each other.

![image](https://user-images.githubusercontent.com/1066253/53373635-e40ca180-390a-11e9-90e7-06a5e742b50f.png)


For a single step I think it may be ok to keep using the dash to do just a single bullet.

For more than one step that are listed next to each other, we can use `1.`